templates/index.json

```diff
@@ -53,7 +53,7 @@
         "heading": {
           "type": "heading",
           "settings": {
-            "heading": "Talk about your brand",
+            "heading": "<p>Talk about your brand</p>",
```

Comment:
**Related**: https://github.com/Shopify/dawn/pull/1625/files#r852183418

[`richtext` input setting](https://shopify.dev/themes/architecture/settings/input-settings#richtext)

`p` tags must be used to wrap the `default` richtext. It will result in an error if it's not:

**`shopify-cli`**

```ruby
11:12:21 ERROR  » update templates/index.json:
  Setting 'heading' is invalid. All top level nodes must be '<p>' tags
```

**Code editor**

![image](https://user-images.githubusercontent.com/37168033/163476901-ba03697b-7da4-4277-9190-8f975a1ec475.png)